Keas.Mvc/ClientApp/components/Equipment/EquipmentTable.tsx

```diff
@@ -0,0 +1,110 @@
+import * as moment from "moment";
+import * as React from "react";
+import ReactTable from "react-table";
+import "react-table/react-table.css";
+import { IEquipment } from "../../Types";
+import ListActionsDropdown from "../ListActionsDropdown";
+
+interface IProps {
+    equipment: IEquipment[];
+    onRevoke?: (equipment: IEquipment) => void;
+    onAdd?: (equipment: IEquipment) => void;
+    showDetails?: (equipment: IEquipment) => void;
+    onEdit?: (equipment: IEquipment) => void;
+}
+
+export default class EquipmentTable extends React.Component<IProps, {}> {
+  public render() {
+      return (
+        <ReactTable
+        data={this.props.equipment}
+        filterable={true}
+        minRows={1}
+        columns = {[
+            {
+                Header: "Serial Number",
+                accessor: "serialNumber",
+                filterMethod: (filter, row) => 
+                    !!row[filter.id] &&
+                    row[filter.id].toLowerCase().indexOf(filter.value) !== -1,
```

Comment:
@laholstege personal preference but I like .contains() instead of indexOf so we don't have remember what -1 means.  Also again I don't think the `!!` is doing anything